### PR TITLE
Fixes #1856 - Better error messaging for unset milestones

### LINKED ISSUE #1856: Better error messaging for unset milestones
Followup on #1852 
Two scenarios, the messages should be along:

* **Anonymous**:  "No assigned status yet" as they can't do anything about it and error will lead to error report. No red color.
* **Logged in**: "Fix me: assign a status"

@zoepage  @miketaylr What do you think about it?

capture d ecran 2017-11-01 a 09 47 57

Once modified. (through developer tools for the mockup)

capture d ecran 2017-11-01 a 09 50 30

### PR DESCRIPTION
**Pull request by marimeireles**
Hi @magsout, I'm not sure if I should ping you or Karl or someone else, so please let me know if I'm pinging the wrong person.

Sorry for the delay I got caught up with school these last few days.
Do you think it's a good solution? Also, did I get the variable names right this time?

Thanks! :)

### CODE REVIEW

**Comment by marimeireles** [comment]
I just saw it failed on the CI tests, this was the test it failed:

```
× firefox on linux 4.4.0-137-generic - Milestones (non-auth) - Missing status error displays (0.201s)
    AssertionError: expected 'No status assigned yet' to equal 'Error: no status for this issue'
    
    A No status assigned yet
    E Error: no status for this issue
    
      at Command.<anonymous>  <tests/functional/milestones-non-auth.js:43:18>
      at Test.Missing status error displays [as test]  <tests/functional/milestones-non-auth.js:41:10>
      at <src/lib/Test.ts:263:51>

```

Should I change this test so it works with the new strings? If yes, should I do this on this issue or should I open a new one?
Thanks!

**Comment by karlcow** [comment]
> Should I change this test so it works with the new strings? If yes, should I do this on this issue or should I open a new one?

@marimeireles Fix the test on this issue. :) That's part of it. The test worked by flagging the change.

**Comment by Anushi1998** [comment]
@marimeireles Also it would be great if you can put `#` before issue number as Git cannot link it now :)

**Comment by karlcow** [comment]
@marimeireles @Anushi1998 The [style we use for commits messages](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md#pull-request-guidelines) (Step 5q) is described in the docs and mildly enforced (when I do not forget).

**Comment by magsout** [comment]
@marimeireles 
>Hi @magsout, I'm not sure if I should ping you or Karl or someone else, so please let me know if I'm pinging the wrong person.

it depends of the stuff. @karlcow or @miketaylr are the specialist of python
@miketaylr and me, are more about JS or CSS stuff. You can ping many people if needed

**Comment by marimeireles** [comment]
>Fix the test on this issue. :) That's part of it. The test worked by flagging the change.
Got it! Thanks, @karlcow. :)

@Anushi1998, yea, you're right! I always forget about that. I'll try to fix it rebasing it. Thanks for the reminder.

@magsout 
>You can ping many people if needed
Okay! :)
I saw your comment on the issue and I'll change the variable names. 
Thanks!

**Comment by Anushi1998** [comment]
@marimeireles I don't know if it would be better, but yes shorter you might consider trying `git commit --amend`

**Comment by marimeireles** [comment]
@Anushi1998 According to [Github's doc](https://help.github.com/articles/changing-a-commit-message/) "If the commit only exists in your local repository and has not been pushed to GitHub, you can amend the commit message with the git commit --amend command."
So I don't think it works when you already pushed.

**Comment by marimeireles** [comment]
That changes everything! I didn't know it.
Thanks for the tip @laghee! I'm totally going to use it! :)

And thanks for sharing the blog posts too. Super useful.

**Comment by Anushi1998** [comment]
@marimeireles Your CircleCI is failing with same error as [#2662 (comment)](https://github.com/webcompat/webcompat.com/pull/2701#issuecomment-436509618).

The possible fix is to update your branch with master and rebase you work :)

**Comment by marimeireles** [comment]
I think now I'm good to go! :)
@magsout r?

**Comment by marimeireles** [comment]
Hi @magsout, I think it's working now.
Please let me know if there's anything I can improve :)

**Comment by marimeireles** [comment]
Thanks for the improvements, @magsout!
(Sorry for the commit mess, I'll fix it once all my changes are approved).

File: webcompat/webhooks/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!doctype html>
```
Comment (karlcow) [inline webcompat/webhooks/index.html]: @marimeireles Is it a mistake in the commit?

Comment (marimeireles) [reply webcompat/webhooks/index.html]: It is. Didn't see it. Thanks.

**Comment by marimeireles** [comment]
@magsout, @karlcow do you think its good enough to merge?
Now I just have to organize my commits, right?

**Comment by magsout** [comment]
@marimeireles yes +1, good job and again thanks.

**Pull request merged**